site/react/useSmartAccountClient.md

```diff
@@ -0,0 +1,98 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: useSmartAccountClient
+  - - meta
+    - name: description
+      content: An overview of the useSmartAccountClient hook
+  - - meta
+    - property: og:description
+      content: An overview of the useSmartAccountClient hook
+  - - meta
+    - name: twitter:title
+      content: useSmartAccountClient
+  - - meta
+    - name: twitter:description
+      content: An overview of the useSmartAccountClient hook
+---
+
+# useSmartAccountClient
+
+The `useSmartAccountClient` hook is used to create a new [`AlchemySmartAccountClient`](/packages/aa-alchemy/smart-account-client/index) attached to either a `LightAccount` or `MultiOwnerModularAccount` contract using the `AlchemySigner`.
+
+## Import
+
+```ts
+import { useSmartAccountClient } from "@alchemy/aa-alchemy/react";
+```
+
+## Usage
+
+<<< @/snippets/react/useSmartAccountClient.tsx
+
+## Parameters
+
+```ts
+import { type UseSmartAccountClientProps } from "@alchemy/aa-alchemy/react";
```

Comment:
does this de-structure into the actual params?

Reply:
no, but it imports it for someone to be able to hover over or have a reference to view the definition. This is something wagmi does too in their docs and it's kinda nice